sdk/monitor/monitor-opentelemetry-exporter/src/export/statsbeat/statsbeatMetrics.ts

```diff
@@ -33,22 +33,26 @@ import {
   NON_EU_CONNECTION_STRING,
   CommonStatsbeatProperties,
   NetworkStatsbeatProperties,
+  VirtualMachineInfo,
+  StatsbeatOptions,
 } from "./types";
 
 const os = require("os");
 
 export class StatsbeatMetrics {
```

Comment:
Renaming this to something else would be useful to quickly understand what is the difference between this one and longInterval one, maybe ShortIntervalStatsbeatMetrics or NetworkStatsbeatMetrics?